profiling/cpu.go

```diff
@@ -14,6 +15,9 @@ var cpuFile *os.File
 // StartCPUProfile enables CPU profiling for the current process. While profiling,
 // the profile will be buffered and written to the file in folder dataDir.
 func StartCPUProfile(dataDir string) error {
+	if cpuFile != nil {
+		return errors.New("cpu profiling is already started")
+	}
 	if cpuFile == nil {
```

Comment:
No longer needed after the previous check and return

Reply:
Indeed! will remove, thank you 🙂